src/linker/implementation_metadata.yaml

```diff
@@ -1,20 +1,3 @@
-# pvs-like
```

Comment:
Starting to remove pvs-like from the codebase. I figure we should not remove it completely (i.e. leave the files in `specifications/` and `steps/`) until R and local spark get implemented.